translator.py

```diff
@@ -49,7 +49,7 @@ def TranslateWord(word):
         translation += "way"
 
     # We only support capital letters at the beginning of the word.  Stomp
```

Comment:
Apparently that last comment was creating a "review", and now I'm adding another comment to the original thread...